core/input/input_map.cpp

```diff
@@ -399,21 +399,25 @@ const HashMap<String, List<Ref<InputEvent>>> &InputMap::get_builtins() {
 	inputs = List<Ref<InputEvent>>();
 	inputs.push_back(InputEventKey::create_reference(Key::LEFT));
 	inputs.push_back(InputEventJoypadButton::create_reference(JoyButton::DPAD_LEFT));
+	inputs.push_back(InputEventJoypadMotion::create_reference(JoyAxis::LEFT_X, -1.0));
```

Comment:
Are we sure that all Controls handle analog input correctly?

Reply:
I still pretty new to Godot Engine's code, but since we check with ui_XXX, should the alt key take the lead if the analog isn't set?

Reply:
What I mean is that this PR adds a specific logic to two nodes just to handle analog input, so I wonder if all other controls can handle it. If not, including axis in default controls can lead to exposing buggy behavior.

Reply:
We got a [discussion]( https://github.com/godotengine/godot/pull/63168#issuecomment-1419268885) about it and I ended up doing a per control implementation since the behavoir of some control node can be quite different from one to another.

That's why I targetted pop_menu and slider, the most common control that should be usable with a gamepad. 

Of course, there is maybe a solution to add it to all control by I'm certainly too inexperienced with the engine to find that solution.

Reply:
Adding it to other Controls would involve testing all of them and tweaking the behavior where necessary.
Good point about commonly used Controls, Tree for example is not used in games at all, so there is no point in fixing it. So I think this PR is enough to cover the most common use-cases.